frontend/src/stores/user-store.ts

```diff
@@ -25,6 +35,50 @@ const initialUserObject = {
 export const useUserStore = defineStore('user', () => {
   const data = useLocalStorage('tba/user', structuredClone(initialUserObject));
 
+  const call = ref(null);
+
+  /**
+   * Initialize store with data required at runtime
+   * 
+   * @param fetch preconfigured function to perform API calls
+   */
+  const init = (fetch: Fetch) => {
+    call.value = fetch;
+  }
+
+  // Init user config if not already available
+  if (!data.value?.settings) {
+    const dj = inject(dayjsKey);
+    const detectedTimeFormat = Number(dj('2022-05-24 20:00:00').format('LT').split(':')[0]) > 12 ? 24 : 12;    
```

Comment:
I think using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/hourCycle will work better?

Reply:
The hourCycle property turned out to not being meant to work for detection. It is just a property already coded in the locale, e.g. the "h23" in "fr-FR-u-hc-h23". The function actually providing time format information for locales is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getHourCycles but thats not yet working in Firefox, unfortunately.